backend/api/appsettings.Development.json

```diff
@@ -14,7 +14,7 @@
     }
   },
   "AllowedHosts": "*",
-  "AllowedOrigins": "http://localhost:3001",
+  "AllowedOrigins": "http://localhost:3000",

```

Comment:
Why? I think we need 3001 for the frontend to be able to connect to Flotilla backend?

Reply:
My bad, i mixed it up with the ISAR port